frontend/src/scenes/session-recordings/components/PanelSettings.tsx

```diff
@@ -37,19 +40,23 @@ export function SettingsBar({
     border: 'bottom' | 'top' | 'all' | 'none'
     className?: string
 }>): JSX.Element {
+    const containerRef = useRef<HTMLDivElement>(null)
     return (
-        <div
-            className={clsx(
-                'flex flex-row w-full overflow-hidden font-light text-xs bg-primary items-center',
-                className,
-                {
-                    'border-b': ['bottom', 'all'].includes(border),
-                    'border-t': ['top', 'all'].includes(border),
-                }
-            )}
-        >
-            {children}
-        </div>
+        <FloatingContainerContext.Provider value={containerRef}>
```

Comment:
only added this floating container context to try out container queries but the positioning of the menu was off so I went a different way...

it does no harm so i haven't bothered removing it